docs/guides/basics/authentication/index.md

```diff
@@ -542,7 +542,7 @@ pack.setUserAuthentication({
 });
 ```
 
-The step's `getOptions` works like [dynamic autocomplete][autocomplete_dynamic], fetching the list of available endpoints and returning a display name and value for each. After the user enters their credentials they will be prompted to select one of the endpoints in a dialog.
+The step's `getOptions` works like [dynamic autocomplete][autocomplete_dynamic], fetching the list of available endpoints and returning a display name and value for each. Note that the returned endpoints must be an object with a name and value field. Further, the value field must be a string that can be parsed by javascript's `new URL(value)` function. After the user enters their credentials they will be prompted to select one of the endpoints in a dialog.
```

Comment:
"with a name and value field" - I think specifically it's `display` and `value`, and it may be better to clarify that.

Reply:
Done